style.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #170606;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+header {
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  background-image: url(Illustration.png);
+  background-repeat: no-repeat;
+  background-size: 200px 300px;
+  background-position: 170px 320px;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 30px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: white;
+  background: #3c3a39;
+}
+
+nav i:hover {
+  color: #bbbab8;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+nav h5:hover {
+  color: #bbbab8;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+section {
```

Comment:
[Optional - but highly recommended]

Later on in the project, actually starting with the next milestone, you will already use other `section` tags, and of course, this will affect your already done styling. That's why is not recommended to style directly the big containers using the tags, but using classes. 
Using classes from the early stages will make your life easier later on.

 - [ ] I suggest you use classes when styling elements and not tags for the exact reasons stated above.